cart-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,14 @@
+server:
+  port: 9096
+spring:
+  application:
+  name: CartService
+  datasource:
+    url: jdbc:mysql://${MYSQL_HOST:localhost}:3306/mydb
+    driverClassName: com.mysql.cj.jdbc.Driver
+    username: ${DB_USER}
```

Comment:
rename db user and password according to another service. You can check it from order-service .yml file

Reply:
It is already the same , 

username: ${DB_USER}
password: ${DB_PASSWORD}

Reply:
No, please check from the 'main' not 'dev'
https://github.com/NashTech-Labs/car-demo/blob/main/order-service/src/main/resources/application.yml